assets/js/common/SuseManagerConfig/SuseManagerConfig.stories.jsx

```diff
@@ -77,4 +88,16 @@ export const WithVeryLongSUMAUrl = {
   },
 };
 
-export const Empty = { args: {} };
+export const Empty = {
+  args: {
+    userAbilities: [{ name: 'all', resource: 'all' }],
+    permitted: ['all:all'],
+  },
+};
+
+export const WithoutPermission = {
```

Comment:
Same naming as before. And better use an empty list, empty strings looks strange